spring-integration-core/src/test/java/org/springframework/integration/handler/MethodInvokingMessageProcessorTests.java

```diff
@@ -583,26 +586,30 @@ public void testPerformanceSpelVersusInvocable() throws Exception {
 		StopWatch stopWatch = new StopWatch("SpEL vs Invocable Performance");
 
 		stopWatch.start("SpEL");
-		for (int i = 0; i < 10000; i++) {
+		int count = 10_000;
+		for (int i = 0; i < count; i++) {
 			processor.processMessage(message);
 		}
 		stopWatch.stop();
 
 		processor = new MethodInvokingMessageProcessor(service, method);
 
 		stopWatch.start("Invocable");
-		for (int i = 0; i < 10000; i++) {
+		for (int i = 0; i < count; i++) {
 			processor.processMessage(message);
 		}
 		stopWatch.stop();
 
+		// Update the parser configuration compiler mode
+		SpelParserConfiguration config = TestUtils.getPropertyValue(processor,
+				"delegate.handlerMethod.EXPRESSION_PARSER.configuration", SpelParserConfiguration.class);
+		new DirectFieldAccessor(config).setPropertyValue("compilerMode", SpelCompilerMode.IMMEDIATE);
```

Comment:
`integerMethod` is not compilable either, because `AnnotatedTestService` is not `public`.